frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel.tsx

```diff
@@ -302,7 +302,7 @@ export const PaymentsInputPanel = (): JSX.Element | null => {
   return (
     <>
       {isPaymentDisabled && (
-        <Box px="1.5rem" pt="2rem" pb="1.5rem">
+        <Box pt="2rem" pb="1.5rem">
```

Comment:
Is this intended? 

as it stretches the inline message to the entire drawer

<img width="609" alt="image" src="https://github.com/opengovsg/FormSG/assets/59867455/2e074bfc-4fd5-4dc9-8c05-94846f510bef">

Reply:
Ps, I accidentally removed this 😓 done in [af4923f](https://github.com/opengovsg/FormSG/pull/6327/commits/af4923fb177bfae5c9385a719260fb2c10ac25b7)